web/components/ImportButton.vue

```diff
@@ -0,0 +1,123 @@
+        <form>
+          <input type="file" @change="readfile" />
+        </form>
+        <div>{{ data.filedata }}</div>
+      </v-card-text>
+
+      <v-divider></v-divider>
+
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn color="green darken-1" text @click="data.dialog = false">
+          Cancel
+        </v-btn>
+        <v-btn color="green darken-1" text @click="ImportScheduler()">
+          Import
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import { defineComponent, inject, reactive } from "@nuxtjs/composition-api";
+import { importScheduler } from "../api/v1/export";
+import SnackBarStoreKey from "./StoreKey/SnackBarStoreKey";
+import SchedulerConfigurationStoreKey from "./StoreKey/SchedulerConfigurationStoreKey";
+import PriorityClassStoreKey from "./StoreKey/PriorityClassStoreKey";
+import StorageClassStoreKey from "./StoreKey/StorageClassStoreKey";
+import PersistentVolumeClaimStoreKey from "./StoreKey/PVCStoreKey";
+import PersistentVolumeStoreKey from "./StoreKey/PVStoreKey";
+import NodeStoreKey from "./StoreKey/NodeStoreKey";
+import PodStoreKey from "./StoreKey/PodStoreKey";
+
+export default defineComponent({
+  setup() {
+    const data = reactive({
+      dialog: false,
+      filedata: "",
+    });
+    const schedulerconfigurationstore = inject(SchedulerConfigurationStoreKey);
+    if (!schedulerconfigurationstore) {
+      throw new Error(`${SchedulerConfigurationStoreKey} is not provided`);
+    }
+    const priorityclassstore = inject(PriorityClassStoreKey);
+    if (!priorityclassstore) {
+      throw new Error(`${PriorityClassStoreKey} is not provided`);
+    }
+    const storageclassstore = inject(StorageClassStoreKey);
+    if (!storageclassstore) {
+      throw new Error(`${StorageClassStoreKey} is not provided`);
+    }
+    const pvcstore = inject(PersistentVolumeClaimStoreKey);
+    if (!pvcstore) {
+      throw new Error(`${PersistentVolumeClaimStoreKey} is not provided`);
+    }
+    const pvstore = inject(PersistentVolumeStoreKey);
+    if (!pvstore) {
+      throw new Error(`${PersistentVolumeStoreKey} is not provided`);
+    }
+    const nstore = inject(NodeStoreKey);
+    if (!nstore) {
+      throw new Error(`${NodeStoreKey} is not provided`);
+    }
+    const pstore = inject(PodStoreKey);
+    if (!pstore) {
+      throw new Error(`${PodStoreKey} is not provided`);
+    }
+
+    const snackbarstore = inject(SnackBarStoreKey);
+    if (!snackbarstore) {
+      throw new Error(`${SnackBarStoreKey} is not provided`);
+    }
+    const setServerErrorMessage = (error: string) => {
+      snackbarstore.setServerErrorMessage(error);
+    };
+    async function ImportScheduler() {
+      Promise.resolve()
+        .then(() => {
+          return importScheduler(JSON.parse(data.filedata));
+        })
+        .then(() => {
+          schedulerconfigurationstore.fetch();
+          priorityclassstore.fetchlist();
```

Comment:
I have no idea to fix these errors.... 
(server is working fine)
```
 ERROR  ERROR in components/ImportButton.vue:98:11                                         00:38:35
TS2532: Object is possibly 'undefined'.
     96 |         })
     97 |         .then(() => {
  >  98 |           schedulerconfigurationstore.fetch();
        |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
     99 |           priorityclassstore.fetchlist();
    100 |           storageclassstore.fetchlist();
    101 |           pvcstore.fetchlist();
    ```

Reply:
`priorityclassstore.fetchlist();`, `storageclassstore.fetchlist();`... also have get the same error😭

>  I think schedulerconfigurationstore.fetch(); is not needed.

Why is `schedulerconfigurationstore.fetch();` not needed? `import` function also import schedulerconfiguration on here.
https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/81/files#diff-a7f49809bda424c7b815cb41c55af4893fcb565763bf4d2b2be137a8d5341de5R224
So, we should reload(fetch) it, shouldn't we?
Thanks!

Reply:
> Why is schedulerconfigurationstore.fetch(); not needed? 

As I said above, when a user wants to see the scheduler config, he will push the gear button and `schedulerconfigurationstore.fetchSelected` will be called then.  (`schedulerconfigurationstore.fetchSelected` will fetch the latest scheduler configuration from API.)
So, creating `schedulerconfigurationstore.fetch` func and calling it here doesn't make sense.

Reply:
Sorry, I was misread...
ok, I will remove `schedulerconfigurationstore.fetch` func.